packages/openapi/api/actions.yaml

```diff
@@ -0,0 +1,84 @@
+openapi: '3.0.3'
+info:
+  title: 'Sofie User Actions API'
+  description: 'The Sofie User Actions API provides paths to allow a device to control a Sofie and query health information'
+  version: '0.0.1'
+  license:
+    name: 'MIT License'
+    url: 'http://opensource.org/licenses/MIT'
+servers:
+  - url: http://localhost:3000/api/v1.0
+    description: Initial release of UserActions API - http - for use during development only!
+tags:
+  - name: userActions
+    description: Provides the ability to control the Sofie application
+paths:
+  /:
+    $ref: 'definitions/sofie.yaml#/resources/index'
+  /system/blueprint:
+    $ref: 'definitions/sofie.yaml#/resources/systemBlueprint'
+  /system/migrations:
+    $ref: 'definitions/sofie.yaml#/resources/migrations'
+  # peripheral devices
+  /devices:
+    $ref: 'definitions/devices.yaml#/resources/devices'
+  /devices/{deviceId}:
+    $ref: 'definitions/devices.yaml#/resources/device'
+  /devices/{deviceId}/action:
+    $ref: 'definitions/devices.yaml#/resources/deviceAction'
+  # blueprint operations
+  /blueprints:
+    $ref: 'definitions/blueprints.yaml#/resources/blueprints'
+  /blueprints/{blueprintId}:
+    $ref: 'definitions/blueprints.yaml#/resources/blueprint'
+  # playlist operations
+  /playlists:
+    $ref: 'definitions/playlists.yaml#/resources/playlists'
+  /playlists/{playlistId}/activate:
+    $ref: 'definitions/playlists.yaml#/resources/activate'
+  /playlists/{playlistId}/deactivate:
+    $ref: 'definitions/playlists.yaml#/resources/deactivate'
+  /playlists/{playlistId}/executeAdLib:
+    $ref: 'definitions/playlists.yaml#/resources/executeAdLib'
+  /playlists/{playlistId}/moveNextPart:
+    $ref: 'definitions/playlists.yaml#/resources/moveNextPart'
+  /playlists/{playlistId}/moveNextSegment:
```

Comment:
Using hyphens ("-") and lowercase in the URLs instead of camelCase seems to be recommended over at [restfulapi.net](https://restfulapi.net/resource-naming/)
```
/playlists/{playlistId}/moveNextSegment -> /playlists/{playlistId}/move-next-segment
```
should we change the URLs to reflect that?